generators/server/templates/src/main/resources/mails/creationEmail.html

```diff
@@ -12,8 +12,8 @@
             Your JHipster account has been created, please click on the URL below to access it:
         </p>
         <p>
-          <a th:href="@{|${baseUrl}/#/reset/finish?key=${user.resetKey}|}"
```

Comment:
OK, I got confused because creationEmail() is not called anywhere and the email for activation has the same link.  There also already exists a reset password email template that uses my "corrected" link value.  I'll review, you are probably right.